/**
Simple motor driver config for Pi 5 grouped outputs
*/
motor_driver:
  ros__parameters:
    cmd_vel_topic: /diffbot/cmd_vel
    pwm_frequency: 50
    left_pin: 18
    right_pin: 19
    max_linear_velocity: 1.0
    max_angular_velocity: 1.0
    safety_timeout: 1.0
